packages/warriorjs-core/src/Unit.js

```diff
@@ -12,17 +12,17 @@ class Unit {
    * @param {number} reward The number of points to reward when killed.
    * @param {boolean} captive Whether the unit is a captive or not.
    */
-  constructor(name, character, maxHealth, reward = null, captive = false) {
+  constructor(name, character, maxHealth, reward = null, hostile = true) {
     this.name = name;
     this.character = character;
     this.maxHealth = maxHealth;
     this.reward = reward === null ? maxHealth : reward;
-    this.captive = captive;
+    this.hostile = hostile;
     this.abilities = new Map();
     this.effects = new Map();
     this.health = maxHealth;
     this.position = null;
-    this.bound = captive;
+    this.bound = !this.hostile;
```

Comment:
This should now be assigned the `bound` argument (let's also move it up to right after `this.hostile` assignment.